clients/drcachesim/scheduler/scheduler_impl.h

```diff
@@ -904,7 +891,7 @@ template <typename RecordType, typename ReaderType> class scheduler_impl_tmpl_t
     stream_status_t
     stop_speculation(output_ordinal_t output);
 
-    stream_status_t
+    virtual stream_status_t
```

Comment:
This is from the user interface.  Added:

    // The external interface lets a user request that an output go inactive when
    // doing dynamic scheduling.

For grouping: there are tradeoffs as this is in an external interface and is grouped next to other interfaces.  It's not pure-virtual so only the dynamic subclass has to override (to avoid all other subclasses from having to override with an identical function returning an error).

Reply:
Update: ended up grouping all the virtual ones together.